Doc/whatsnew/3.11.rst

```diff
@@ -74,8 +74,11 @@ New syntax features:
 
 New typing features:
 
-* :pep:`673`: ``Self`` Type.
-  (Contributed by James Hilton-Balfe and Pradeep Kumar in :issue:`30924`.)
+* :pep:`646`: Variadic generics.
+* :pep:`655`: Marking individual TypedDict items as required or potentially-missing.
```

Comment:
```suggestion
* :pep:`655`: Marking individual TypedDict items as required or potentially missing.
```

Pet peeve: nnecessary hyphen https://prowritingaid.com/grammar/1000138/How-to-use-hyphen-after-adverbs

Reply:
Oh whoops that's the verbatim title of @davidfstr's accepted PEP, so I guess it's too late for me to pick that fight 😆 as you were